hummingbot/market/market_base.pyx

```diff
@@ -194,6 +194,9 @@ cdef class MarketBase(NetworkIterator):
         order_size_quantum = self.c_get_order_size_quantum(trading_pair, amount)
         return (amount // order_size_quantum) * order_size_quantum
 
+    def supported_order_types(self):
+        pass
```

Comment:
This should raise NotImplementedError, st. we know it hasn't been implemented on the subclass.

Reply:
Done!